core/app.js

```diff
@@ -28,6 +28,11 @@ if(ENV.HLRDESK_DEV) {
   app.use(require('./app_modules/koa-sassy').Sassy);
 }
 
+process.on("uncaughtException", function(err) {
+  email.serverCrash(err.message, err.stack);
```

Comment:
We'll probably want these output in the logs, too. Maybe instead of calling `process.exit(1)` just throw the error again?